### ISSUE TITLE
Need to be able to make a digital signature.

### ISSUE THREAD

**Issue by jaytolentino** [AloeCare/aloe-app#1]
As per https://github.com/AloeCare/aloe-design/issues/8, someone filling out the intake forms needs to be able to do an electronic signature.

Here's the most applicable resource on it:
- [Can e-signatures be used under HIPAA rules?](http://www.hipaajournal.com/can-e-signatures-be-used-under-hipaa-rules-2345/)

Other less important sources:
- [Are digital signatures legally binding](https://www.formstack.com/blog/2014/digital-signatures-legally-binding/)
- [How do I turn a checkbox click into a digital signature?](https://www.quora.com/How-do-I-turn-a-checkbox-click-into-a-digital-signature)
- [Sign Here: Electronic Signatures and the In-House Counsel](http://www.acc.com/vl/public/ACCDocketArticle/loader.cfm?csModule=security/getfile&pageid=1391627&page=/legalresources/resource.cfm&qstring=show=1391627&title=Sign%20Here%20Electronic%20Signatures%20and%20the%20Inhouse%20Counsel&recorded=1)

**Comment by jaytolentino** [comment]
@truongmm could you start by doing some research on this? Perhaps looking at 3rd party options.

**Comment by truongmm** [comment]
@jaytolentino Yep yep. Let's go over the tasks tonight since I finished the routing!

**Comment by jaytolentino** [comment]
@truongmm yay sounds good!
